### ISSUE TITLE
Prev/Play/Next buttons are misaligned

### ISSUE THREAD

**Issue by RomRider** [kalkih/mini-media-player#16]
Fantastic work with this Lovelace card @kalkih !!
In the latest version there is an issue with the placement of the Prev/Play/Next buttons when the title of what is currently playing doesn't take more than the whole width of the card.

Please see this image:
![image](https://user-images.githubusercontent.com/21064206/46816145-9e48a580-cd7c-11e8-9610-27bf23d0984d.png)

If the title is long enough, it works as expected:
![image](https://user-images.githubusercontent.com/21064206/46816521-73128600-cd7d-11e8-9796-f555b7982d9d.png)

My current config is:
```yml
      - type: custom:mini-media-player
        entity: media_player.sonos_connect
        name: Sonos Connect
        artwork_border: true
        group: true
        hide_power: true
        hide_volume: true
        short_info: true
        scroll_info: true
        show_progress: false
```

Thanks a lot for your help.
Cheers

**Comment by kalkih** [comment]
Thank you for reporting this, thee issue was introduced with the `hide_info` feature in v0.8.4.

I've fixed this  with 86107ddac3d1313af5276e21242b64abe0554127

**Comment by RomRider** [comment]
Thanks a lot!
